Models/NewYorkCrime.cs

```diff
@@ -14,28 +14,37 @@ namespace NewYorkCityCrimeData
     using System.Globalization;
     using Newtonsoft.Json;
     using Newtonsoft.Json.Converters;
+    using System.Runtime.Serialization;
 
-    public partial class CrimeData
+    [DataContract]
+    public partial class NewYorkCrime
```

Comment:
I'd leave "NewYork" out of variable names, because this can be reapplied to other geographies.